CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
 
 project(Hyrise)
 
+enable_testing()
```

Comment:
When looking for documentation (which I haven't found so far), I stumpled over this post that states that including GoogleTest automatically calls `enable_testing()` (see: https://stackoverflow.com/a/50476125/1147726). Do we still need it here?

Reply:
The linked post mentions `include(CTest)` not `include(GoogleTest)`